### ISSUE TITLE
Some questions about training process

### ISSUE THREAD

**Issue by modyu-liu** [JyChen9811/FaithDiff#9]
Hi, thank you for your excellent work. I’m a bit confused about the training process and would appreciate some clarification.

I’m not sure if I understood correctly — did you enable almost all SDXL modules for training in the second stage?

And, in the paper, you mentioned using two 80G A800 GPUs with 512×512 crops and a batch size of 256 — may I ask how that was achieved?

**Comment by JyChen9811** [comment]
Hi, here is the accelerate config of the paper: gradient accumulation steps are set to be 8, and two GPUs (num_processes) are used.

```
compute_environment: LOCAL_MACHINE
debug: false
deepspeed_config:
  gradient_accumulation_steps: 8
  gradient_clipping: 1.0
  offload_optimizer_device: none
  offload_param_device: none
  zero3_init_flag: false
  zero_stage: 2
distributed_type: DEEPSPEED
downcast_bf16: 'no'
enable_cpu_affinity: false
machine_rank: 0
main_training_function: main
mixed_precision: fp16
num_machines: 1
num_processes: 2
rdzv_backend: static
same_network: true
tpu_env: []
tpu_use_cluster: false
tpu_use_sudo: false
use_cpu: false
main_process_port: 7399
```

**Comment by JyChen9811** [comment]
In addition, we enable almost all SDXL modules for training in the second stage.
